CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+[![License MIT](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/org-formation/org-formation-cli/issues)
+
+## How to contribute to Org-formation
+
+### **Did you find a bug?**
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/org-formatio/org-formation-cli/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/org-formatio/org-formation-cli/issues/new).
+  Be sure to include a **title and clear description**, as much relevant information as possible, and a **project sample**
```

Comment:
i would drop "as much relevant information as possible, and a **project sample** demonstrating the expected behavior that is not occurring.". this will be part of the GH issue template

Reply:
done